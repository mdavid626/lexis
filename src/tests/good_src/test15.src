/' Program 15: strings '/

function neco ( s as string) as string
	print s; "\n";
	s = "Ahoj, nazdarek"
	print s; "\n";
	s = "Ahoooj"
	return s
end function

scope
dim s as string
dim res as string
s = "Scope"

print s; "\n";
res = neco(s)
print s; "\n";
print res; "\n";
end scope
