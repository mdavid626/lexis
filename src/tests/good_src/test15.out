Scope
Scope
Ahoj, nazdarek
Scope
Ahoooj
